packages/trimerge-sync/src/TrimergeClient_2Users.test.ts

```diff
@@ -297,6 +304,9 @@ Array [
         "userId": "a",
```

Comment:
Should we add a client 2 subscriber and assert that its origin is correct?

Reply:
good call, it's not correct! :)

Reply:
okay, now they always say 'local' which doesn't feel quite right, but I'm not sure what is the expected behavior for ClientList state changes?